### ISSUE TITLE
Bonsai trees not working

### ISSUE THREAD

**Issue by jamzam90** [TheLimePixel/Modern-Skyblock-3#158]
I am unable to plant saplings in bonsai trees as of pack version 3.1.5

**Comment by Budda523** [comment]
Have the same problem.
My old bonsai pots are still working, but i can't place a sap in an empty hopping bonsai pot.

**Comment by b0bst3r** [comment]
In MSB 3.1.5- To plant a sapling in a bonsai pot you have to right-click the sides of a bonsai pot.  Dave has fixed this and this is from the changelog.

**Comment by GilesVolmir** [comment]
My bonsai pots are doing the same thing: old ones working but new ones can't be planted.  Cannot replant broken and re-placed pots either.
I tried right clicking the side of the pot with the sapling instead of the pot, but that didn't do anything.  I tried it in several spots on the model and all sides of the pot and then tried from the bottom and still nothing.  I then also tried clicking on all 4 sides with the sapling in my off-hand because I noticed a changelog related to that but that didn't work either.
I have tried with both bonsai pots and hopping bonsai pots.
I just double checked and I have version 3.1.5
Am I missing something?
